torchdynamo/convert_frame.py

```diff
@@ -101,6 +103,27 @@ def _fn(*args, **kwargs):
     return _fn
 
 
+def has_tensor_in_frame(frame):
+    # Check if the passed arguments are of type Tensor
+    for value in frame.f_locals.values():
+        if isinstance(value, torch.Tensor):
```

Comment:
We should also check for torch.nn.Module. 

We should also check for `List[Tensor]`, `Dict[str, Tensor]`, etc.  Perhaps have small helper that walks data structures.